src/wallet/rpcwallet.cpp

```diff
@@ -850,7 +885,9 @@ UniValue getbalance(const JSONRPCRequest& request)
             "Note that the account \"\" is not the same as leaving the parameter out.\n"
             "The server total may be different to the balance in the default \"\" account.\n"
             "\nArguments:\n"
-            "1. \"account\"        (string, optional) DEPRECATED. The selected account, or \"*\" for entire wallet. It may be the default account using \"\".\n"
+            "1. \"account\"        (string, optional) DEPRECATED. This argument will be removed in V0.18. \n"
+            "                     To use this deprecated argument, start dashd with -deprecatedrpc=accounts.\n"
+            "                     The selected account, or \"*\" for entire wallet. It may be the default account using \"\".\n"
```

Comment:
It looks like all the params should be marked as deprecated, see https://github.com/bitcoin/bitcoin/pull/12953/files#diff-63c7710d72381cffe692175dd70e47f1891bf75c3e09249ed826f9df3917d02dR870